app/AppKernel.php

```diff
@@ -20,7 +20,8 @@ public function registerBundles()
             new Symfony\Bundle\AsseticBundle\AsseticBundle(),
             new Doctrine\Bundle\DoctrineBundle\DoctrineBundle(),
             new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
-            new AppBundle\AppBundle(),
+            new Knp\Bundle\PaginatorBundle\KnpPaginatorBundle(),
+            new AppBundle\AppBundle()
```

Comment:
The official reasoning for this is that when you add a new element to the array, you just need to add that element, not the element and a comma before it. This makes the diff operation cleaner for the pull request.